### PR TITLE
Fix png download problem in ios

### PR DESCRIPTION
**Pull request by MahboobehMohammadi**

### CODE REVIEW

File: sass/styles/chart.scss [hunk 1]
```diff
@@ -978,4 +978,8 @@ $spot-color: #0298d3;
             margin-left: -2px !important;
         }
     }
+    .ciq-chart.ciq-screenshot .stx-btn-panel {
+        //CSS images cause cross origin security error when export png
+        display: none !important;
```
Comment (brucebinary) [inline sass/styles/chart.scss]: nitpick: you can put this under `.ciq-chart.ciq-screenshot` in line 913

Comment (MahboobehMohammadi) [reply sass/styles/chart.scss]: @brucebinary I want this only for mobiles

Comment (brucebinary) [reply sass/styles/chart.scss]: @MahboobehMohammadi ah I see. Doesn't hurt to put it at both places I think. I won't miss it not being in the desktop screenshot

Comment (MahboobehMohammadi) [reply sass/styles/chart.scss]: @brucebinary yup. it will. because it hides the pngs for a sec and it doesnt looks good in desktop.

**Pull request merged**